### PR TITLE
New abbrs and last changes before dictionary

### LINKED ISSUE #14: i, j  and k / single letter abbrevs
I noticed that in the latest version i, j  and k are missing. I think "i" is used a lot and "j" also helpful. "k" should be in the list because "v" is (as in key and value)

Possibyl it could be an idea to move all single letter abbrevs into their own section

### LINKED ISSUE #19: "function", "vertical"
"function", current entry:

> 🟢 f / func • function

But in Rust and laptop keyboards, it's `fn`. I've also seen it in code in other languages as an identifier where `fn` isn't a keyword.

---

"vertical", current entry:

> 🟢 ver • version / vertical

I don't think the green circle is valid for both meanings, specifically not for "vertical". According to [this page](https://www.allacronyms.com/vertical/abbreviated), "vert." is much more popular than "ver." for "vertical".

---

> 🟢 num • number

"num" also stands for "number of" (count) as in `numElements`.

---

> 🟢 e.g. • example

This seems unnecessary in this list. And "e.g." isn't just "example", but "for example". Isn't "ex." the abbreviation for "example"?

---

Shouldn't some abbreviations be all-caps?

> 🟢 pr • pull request

### PR DESCRIPTION
**Pull request by T1xx1**
Add discussed abbrs 
- [x] #31 
- [x] #32 
- [x] #33 
- [x] #40
After this pull, I'm closing these issues.

Some suggestions before the dictionary.
1. Clear the issues. Close #14 as included in dictionary and acronym suggested by @oood. Close #17, #19, #20, #29.
2. I think we should open the GitHub projects by adding a list of issues in order of how we should resolve them however we will end up having thousands of issues without resolving them.
3. Add all the new dictionary commits in a new branch before creating the organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Colored circles before the abbr mean the degree of recommendation:
 -  delete • 🟢 del
 -  delta time • 🟡 dt { calculations }
 -  descriptor • 🟢 dsc
+-  description • 🟢 dsc
```
Comment (kisvegabor) [inline README.md:104]: Finally isn't "desc" seemed better?

File: README.md [hunk 2]
```diff
@@ -273,13 +275,13 @@ Colored circles before the abbr mean the degree of recommendation:
 -  standard • 🟢 std
 -  standard input output • 🟢 stdio
 -  string • 🟢 str
--  synchronous • sync
+-  synchronous • 🟢 sync
 -  synchronize • 🟢 sync
```
Comment (kisvegabor) [inline README.md:279]: We need only the noun "synchronization", don't we?

Comment (T1xx1) [reply README.md:279]: Yes, I had this commit on local and we discussed nouns after fixing it right now.

**Comment by kisvegabor** [comment]
I think it's not critical to fix pending abbreviations. We can continue their discussions when the we have the "raw database".

**Comment by oood** [comment]
> Clear the issues. Close i, j and k / single letter abbrevs #14 as included in dictionary and acronym suggested by @oood. Close Add 'chr' to character #17, "function", "vertical" #19, Update README.md #20, pub, priv, lat and lng added or updated #29.


Considering that the project's positioning is to develop into a dictionary, then I support its inclusion. 

Of course, we don't need to include all 26 letters, only those that are most commonly used.

> I think we should open the GitHub projects by adding a list of issues in order of how we should resolve them however we will end up having thousands of issues without resolving them.

Agreed, it makes things easier, especially now that there are multiple to-dos and need a place to record the to-do list.

Would you like to help to see how to enable [this](https://github.com/kisvegabor/abbreviations-in-code/projects)? @kisvegabor 

> Add all the new dictionary commits in a new branch before creating the organization.
> I think it's not critical to fix pending abbreviations. We can continue their discussions when the we have the "raw database".

Sooner or later we all need to add them into the database.

**Comment by T1xx1** [comment]
> Sooner or later we all need to add them into the database.

Yes, thought this before the org project.

**Comment by kisvegabor** [comment]
> I think we should open the GitHub projects by adding a list of issues in order of how we should resolve them however we will end up having thousands of issues without resolving them.

What is the advantage of "Projects" compared to Issue labels?

**Comment by T1xx1** [comment]
> > I think we should open the GitHub projects by adding a list of issues in order of how we should resolve them however we will end up having thousands of issues without resolving them.
> 
> What is the advantage of "Projects" compared to Issue labels?

No just to list things in general. As I think you saw I opened #51 because I needed a place to list. Having labels it's not going to order how we are going to proceed with issues and prs.

**Comment by oood** [comment]
> > > I think we should open the GitHub projects by adding a list of issues in order of how we should resolve them however we will end up having thousands of issues without resolving them.
> > 
> > 
> > What is the advantage of "Projects" compared to Issue labels?
> 
> No just to list things in general. As I think you saw I opened #51 because I needed a place to list. Having labels it's not going to order how we are going to proceed with issues and prs.

Yes, and members of this repository/org can edit it, that todo list (#51) I cannot edit, because that is a personal issue (wait... "_personal issue_"? that sounds weird 🤦 I didn't mean it that anyway.... I just don't know how to describe it, sorry).



> What is the advantage of "Projects" compared to Issue labels?

Not sure if you've used Trello for that (I used to work with a non-profit organization that used the platform to coordinate the daily work of volunteers from all over the world), it's similar to that. for team collaboration, sharing ideas, recording progress, setting to-dos, basically it's visual based and easy for people to see what to do next.

GitHub basically copied the functionality of Trello in my opinion. But their [documentation](
https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) is so bad that most people don't understand what this feature does.

I'll show you a Trello narration video, maybe you'll know what it is. 
https://youtu.be/AyfupeWS0yY

**Comment by oood** [comment]
> GitHub basically copied the functionality of Trello in my opinion.

I need to correct that Trello is more mature and has more features than GitHub's Projects. But enough for a to-do list. It can't replace the collaboration and productivity of Trello, that's for sure.

**Comment by T1xx1** [comment]
> Yes, and members of this repository/org can edit it, but that todo list 
(https://github.com/kisvegabor/abbreviations-in-code/issues/51) I cannot edit, because that is a personal issue (wait... "personal issue"? that sounds weird 🤦 I didn't mean it that anyway.... I just don't know how to describe it, sorry).

HAHAHAHAHAH

> > GitHub basically copied the functionality of Trello in my opinion.
> 
> I need to correct that Trello is more mature and has more features than GitHub's Projects. But enough for a to-do list. It can't replace the collaboration and productivity of Trello, that's for sure.

The only advantage GitHub has it being in the GitHub environment without going anywhere else.

**Comment by oood** [comment]
> The only advantage GitHub has it being in the GitHub environment without going anywhere else.

Yes, in an organization, even a project can be applied to all repositories in the organization. This seamless integration and centralized management make things easier.

**Pull request merged**